GameDataParser/Parsers/AchieveParser.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections.Generic;
+using System.Xml;
+using GameDataParser.Crypto.Common;
+using GameDataParser.Files;
+using Maple2Storage.Types.Metadata;
+
+namespace GameDataParser.Parsers
+{
+    class AchieveParser : Exporter<List<AchieveMetadata>>
+    {
+        public AchieveParser(MetadataResources resources) : base(resources, "achieve") { }
+        protected override List<AchieveMetadata> Parse()
```

Comment:
Can you leave a blank line in between here?

Reply:
you mean between 11 and 12 right? okie

Reply:
Yeah